CHANGELOG.md

```diff
@@ -1,5 +1,57 @@
 # [Versions](https://mui.com/versions/)
 
+## 7.0.0
+
+<!-- generated comparing v7.0.0-rc.0..master -->
+
+_Mar 26, 2025_
+
+🎉 We're happy to announce the stable release of v7.
+
+- Read the blog post [Material-UI v7 stable release](https://mui.com/blog/material-ui-v7-is-here/).
```

Comment:
```suggestion
- Read the blog post - [Material UI v7 is here](https://mui.com/blog/material-ui-v7-is-here/).
```

or 

```suggestion
- Read the [blog post](https://mui.com/blog/material-ui-v7-is-here/) about the announcement.
```

Either use the actual name of the post, or describe it. Also, Material-UI -> Material UI

Reply:
I'll go with the latter one.